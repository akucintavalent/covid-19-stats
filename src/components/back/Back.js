import { useNavigate } from 'react-router-dom';

const Back = () => {
  const navigate = useNavigate();
  return (
    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
    <div
      onKeyDown={() => {}}
      type="button"
      onClick={() => navigate(-1)}
      className="stats"
    >
      &nbsp;&nbsp;&nbsp;
      <i className="fas fa-chevron-left" />
      &nbsp;&nbsp;&nbsp;
    </div>
  );
};

export default Back;
