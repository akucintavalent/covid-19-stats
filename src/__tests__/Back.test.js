import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom';
import Back from '../components/back/Back';

test('back button snapshot', () => {
  const tree = renderer
    .create(<BrowserRouter><Back /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
