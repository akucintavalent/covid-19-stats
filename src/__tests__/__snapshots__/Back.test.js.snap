// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`back button snapshot 1`] = `
<div
  className="stats"
  onClick={[Function]}
  onKeyDown={[Function]}
  type="button"
>
     
  <i
    className="fas fa-chevron-left"
  />
     
</div>
`;
